web-ui/src/components/admin/EditPDL.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useRef, useState } from "react";
+import Avatar from "@material-ui/core/Avatar";
+import Button from "@material-ui/core/Button";
+
+import "./EditPDL.css";
+
+const EditPDL = ({ onDeselect, onEdit, onSelect, profile }) => {
+  const { image_url, name, pdl, role } = profile;
+  const [disabled, setDisabled] = useState(true);
+  const inputRef = useRef();
+
+  let image = image_url ? image_url : "https://i.imgur.com/TkSNOpF.jpg";
```

Comment:
Is this image a placeholder, will we add an image to public?

Reply:
Good catch. I would include this image in our app's static resources instead of referring to the imgur one. Better yet, I'd reach out to Brandon Kelly and see if he can make you one for this.